docs/samples/calling/app.js

```diff
@@ -272,38 +322,7 @@ function userSession() {
 }
 
 function createDevice() {
```

Comment:
do we need this function any more ? why to all this function when we already would have the Callling.callingClinet after init 

we should maintain global names for callingClient in the app.js file , I saw in few places where we were creating client multiple times

Reply:
yes this function is required as this is invoked when we click on register button to trigger mobius registration